src/main/java/com/hotjava/app/hotjava/dao/VehicleDAOStub.java

```diff
@@ -17,14 +17,18 @@
 
     @Override
     public Vehicle save(Vehicle vehicle) throws Exception {
-        allVehicles.put(vehicle.getVehicleSubmissionID(), vehicle);
+        try {
+            allVehicles.put(vehicle.getSubmissionID(), vehicle);
+        }
+        catch (Exception e) {
+            // TODO add exception handling
+        }
         return vehicle;
     }
 
     @Override
     public List<Vehicle> fetchAll() {
-        List<Vehicle> returnVehicles = new ArrayList(allVehicles.values());
-        return returnVehicles;
+        return new ArrayList<>(allVehicles.values());
```

Comment:
Optimized function so that it only took one line, returning an arraylist of objects returned from the allVehicles list.

Reply:
Implemented:
https://github.com/coffee-ns/hotjava/commit/b60898f3141483b2a7f2d82b1df61f3563f860b5